acquisitions/src/main/resources/thunderjet/mod-gobi/features/gobi-api-tests.feature

```diff
@@ -27,3 +27,33 @@ Feature: mod-gobi api tests
     When method POST
     Then status 201
     And match responseHeaders['Content-Type'][0] == 'application/xml'
+    * def poLineNumber = /Response/PoLineNumber
+
+  Scenario: Check the order to approved
+    Given path '/orders/composite-orders'
+    And headers headers
+    And param query = 'poNumber==*' + poLineNumber.split('-')[0]+'*'
+    When method GET
+    Then status 200
+    And match response.purchaseOrders[0].approved == true
+
+  Scenario: Check fields of the order to match with requested data
+    Given path '/orders/order-lines'
+    And param query = 'poLineNumber=="*' + poLineNumber + '*"'
+    And headers headers
+    When method GET
+    Then status 200
+    And match $.poLines[0].checkinItems == false
+    And match $.poLines[0].contributors[0].contributor == 'ANDERSON, KENNETH, 1956-'
+    And match $.poLines[0].cost.listUnitPriceElectronic == 14.95
+    And match $.poLines[0].cost.poLineEstimatedPrice == 14.95
+    And match $.poLines[0].cost.currency == 'USD'
+    And match $.poLines[0].details.productIds[0].productId == '9780817913465'
+    And match $.poLines[0].details.productIds[0].qualifier == '(electronic bk.)'
+    And match $.poLines[0].orderFormat == 'Electronic Resource'
+    And match $.poLines[0].poLineNumber == poLineNumber
+    And match $.poLines[0].poLineNumber == poLineNumber
+    And match $.poLines[0].publisher == 'HOOVER INSTITUTION PRESS,'
+    And match $.poLines[0].publicationDate == '2012.'
+    And match $.poLines[0].titleOrPackage == 'LIVING WITH THE UN[electronic resource] :AMERICAN RESPONSIBILITIES AND INTERNATIONAL ORDER.'
+    And match $.poLines[0].vendorDetail.referenceNumbers[0].refNumber == '99952919209'
```

Comment:
Please check acquisition unit field also , 
(first need to check whether Gobi organization have acq units already assigned if not please assign first. )
Steps:-
1:-create a test acquisition unit.
2:-assign to the Gobi organization
3:-create order via gobi
4:-check order having same acq_unit by Poline number.